log/logger.go

```diff
@@ -195,5 +195,9 @@ func (f *consoleLogFormatter) Format(entry *logrus.Entry) ([]byte, error) {
 		entry.Message = strings.Join(msg, " ")
 		delete(entry.Data, "objects")
 	}
+	if stringObjects, ok := entry.Data["stringObjects"].([]string); ok {
```

Comment:
If `stringObjects` is set, we overwrite the `entry.Message`. Should `stringObjects` be exclusive from `objects`?
In any case, after the changes in this PR, `objects` field is unused in the codebase. Should we maintain its particular formatter implementation?

Reply:
Good spot! I've removed the original `objects` formatter from the logger in 2e9e38701f46676fb5593e9d963c40f7e117766f